src/platformdirs/unix.py

```diff
@@ -93,6 +113,7 @@ def user_state_dir(self) -> str:
             path = os.path.expanduser("~/.local/state")
         return self._append_app_name_and_version(path)
 
+    # TODO: As per XDG spec, logs should be placed under XDG_STATE_HOME
```

Comment:
Please no todos :+1:

Reply:
Forgot to remove. Will file another issue for it, if I remember.